src/applications/mhv/secure-messaging/components/PrintMessageThread.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getThread } from '../actions';
+import MessageThreadItem from './MessageThread/MessageThreadItem';
+
+const PrintMessageThread = props => {
+  const { messageId } = props;
+  const dispatch = useDispatch();
+  const messageThread = useSelector(state => state.message.messages);
+
+  useEffect(
+    () => {
+      if (messageId) {
+        dispatch(getThread(messageId));
+      }
+    },
+    [dispatch, messageId],
+  );
+
+  const messageThreadList = () => {
+    return (
+      <div className="message-thread-list">
+        {messageThread.map((m, i) => {
+          return (
+            <div key={i}>
+              <MessageThreadItem message={m} printView="print view" />

```

Comment:
"print view" should be a constant instead of a string literal in both places it's used.

Reply:
No problem. just changed it.